get_ns.py

```diff
@@ -10,14 +10,24 @@ def is_active_line(line):
     """
     return any(ts in line.decode() for ts in TEST_STRINGS)
 
+def is_active_ns(line):
+    return (ts in line.decode() for ts in TEST_STRINGS)
```

Comment:
You lost the `any` function here!

This makes a huge difference in the meaning - in practice, it always returns true.

Reply:
on purpose
I want to return the ns servers